docs/guides/nrfconnect_examples_configuration.md

```diff
@@ -0,0 +1,136 @@
+# Configuring nRF Connect SDK examples
+
+When you tried to build nRF Connect SDK example applications like
+[lock-app](../../examples/lock-app/nrfconnect/README.md),
+[lighting-app](../../examples/lighting-app/nrfconnect/README.md) or
+[pigweed-app](../../examples/pigweed-app/nrfconnect/README.md), you could notice
+that some default configuration is used then. Probably after testing an example
+solution you may want to change something in the application configuration or
+add new functionalities to build your own application based on sample. This page
+contains information about how to configure an application in different ways and
+also sheds some light on the configuration structure to allow you understand
+building process better.
+
+<hr>
+
+## Configuring application
+
+Changing default application configuration might be done in the many ways and
+you should choose the one that suits you, depending on your needs. You are
+allowed to change configuration temporarily, what will be useful for testing
+impact of changes on the application behavior, but if you want to develop own
+application, probably you will be interested in making permanent changes.
+
+<hr>
+
+### Temporary configuration
+
+Temporary solution is basing on editing zephyr/.config, which is storing all
+configuration options from the whole application generated as a result of
+building process. After cleaning build and deleting this file, all changes are
+gone, so this is not possible to save changes permanently this way.
+
+First thing that needs to be done is bulding the application (see:
+[Building nRF Connect examples](TODO:) for more details), by typing following
+command in the example directory:
+
+        # <board_name> should be replaced with the actual target board name
+        $ west build -b <board_name>
+
+Next, run the terminal-based interface called menuconfig, by typing command:
+
+        $ west build -t menuconfig
+
+You should be able to see menuconfig terminal window, where you can navigate
+using arrow keys and other keys, whose functions are described in the bottom of
+the window. You can make desired changes following terminal instructions and
+after that press `Q` to save and quit.
+
+At this point configuration changes are applied to the output file and it can be
+flashed to the device.
+
+<hr>
+
+### Permanent configuration
+
+Permanent solution is basing on changing configuration Kconfig files used as
```

Comment:
In this chapter, I'm lacking information on where is the KConfig and helper overlays. As well how to enable them e.g. release.conf.

Reply:
If it comes to ovelays - ok, I will add it. Did you mean prj.conf or Kconfig file? Prj.conf location is described below and Kconfig location doesn't seem to me something interesting for the user, as he/she doesn't need to edit it.

Reply:
Note that I described using release.conf overlay in the Building section of the examples.